lua/rustaceanvim/init.lua

```diff
@@ -34,15 +34,23 @@
 --- 'expandMacro' - Expand macros recursively.
 --- 'moveItem {up|down}' - Move items up or down.
 --- 'hover {actions|range}' - Hover actions, or hover over visually selected range.
---- 'explainError' - Display a hover window with explanations form the Rust error index.
----                  Like |vim.diagnostic.goto_next|, |explainError| will cycle diagnostics,
----                  starting at the cursor position, until it can find a diagnostic with
----                  an error code.
---- 'renderDiagnostic' - Display a hover window with the rendered diagnostic,
+--- 'explainError {cycle?|current?}' - Display a hover window with explanations form the Rust error index.
+---                  If cycle option:
+---                    Like |vim.diagnostic.goto_next|, |explainError| will cycle diagnostics,
+---                    starting at the cursor position, until it can find a diagnostic with
+---                    an error code.
+---                  If current option
+---                    Searches for diagnostics only in the current cursor line
+---                  Defaults to 'explainError cycle', if called without an argument 
+--- 'renderDiagnostic {cycle?|current?}' - Display a hover window with the rendered diagnostic,
 ---                      as displayed during `cargo build`.
----                  Like |vim.diagnostic.goto_next|, |renderDiagnostic| will cycle diagnostics,
----                  starting at the cursor position, until it can find a diagnostic with
----                  rendered data.
+---                  If cycle option:
+---                    Like |vim.diagnostic.goto_next|, |renderDiagnostic| will cycle diagnostics,
+---                    starting at the cursor position, until it can find a diagnostic with
+---                    rendered data.
+---                  If current option
+---                    Searches for diagnostics only in the current cursor line
+---                  Defaults to 'renderDiagnostic cycle', if called without an argument 
```

Comment:
Same suggestion as in the readme (only here, use `|` instead of backticks, because I generate vimdoc from the Lua docs).